src/composables/useConversationInfo.js

```diff
@@ -43,7 +43,7 @@ export function useConversationInfo({
 	})
 
 	const hasLastMessage = computed(() => {
-		return !!Object.keys(Object(item.value?.lastMessage)).length
```

Comment:
@DorraJaouad don't we need the old check for desktop client compatibility with old servers?

Reply:
True